master/main.py

```diff
@@ -0,0 +1,17 @@
+######################
+# Main functionality #
+######################
+
+import smbus
+import module.scheduler as scdr
+
+def main():
+    scheduler = scdr.Scheduler()
+    scheduler.enqueue_smoothie()
```

Comment:
I'm assuming the plan is to put `enqueue_smoothie` in an ISR? And have this main function loop forever?